concent_api/requirements.lock

```diff
@@ -43,6 +44,7 @@ kiwisolver==1.0.1
 kombu==4.2.1
 lru-dict==1.1.6
 matplotlib==2.2.2
+Middleman-Protocol==0.8.0+dev194.gce34fd0
```

Comment:
We want to have a path in the .lock file too.

We definitely don't want to hard-code the commit ID here. It's always going to be out of date because you don't know the commit ID until you commit...

Reply:
right, done